ReleaseHistory.md

```diff
@@ -18,6 +18,9 @@
 - UER => Eliminate unhandled exceptions in rule.
 - UEE => Eliminate unhandled exceptions in engine.
 
+## v4.3.11 Unreleased
+- BRK: Disable `SEC101/047.CratesApiKey`. There is no clear way to do authentication. The current duyanmic validator return OK given any regex-matched secret.
```

Comment:
>duyanmi [](http://example.com/codeflow?start=98&length=7)

dynamic

Reply:
I'd say "Current dynamic validator returns status code 200 to all tokens. No API endpoint seems to returns different status codes to valid vs invalid API Keys."